docs/website/docs/overview/configure.md

```diff
@@ -190,6 +190,7 @@ Options here are mostly used for debugging and testing `odo` behavior.
 | `DEVFILE_PROXY`            | Integration tests will use this address as Devfile registry instead of `registry.stage.devfile.io`                                                                                                                                                                                                                                                                             | v3.0.0-beta3  | `my-registry.example.com`       |
 | `TELEMETRY_CALLER`         | Caller identifier passed to [telemetry](https://github.com/redhat-developer/odo/blob/main/USAGE_DATA.md). Case-insensitive. Acceptable values: `vscode`, `intellij`, `jboss`.                                                                                                                                                                                                  | v3.1.0        | `intellij`                      |
 | `ODO_TRACKING_CONSENT`     | Useful for controlling [telemetry](https://github.com/redhat-developer/odo/blob/main/USAGE_DATA.md). Acceptable values: `yes` ([enables telemetry](https://github.com/redhat-developer/odo/blob/main/USAGE_DATA.md) and skips consent prompt), `no` (disables telemetry and consent prompt). Takes precedence over the [`ConsentTelemetry`](#preference-key-table) preference. | v3.2.0        | `yes`                           |
-| `ODO_EXPERIMENTAL_MODE`    | Whether to enable experimental features. See [Experimental Mode](../user-guides/advanced/experimental-mode) for more details. Acceptable values: boolean values<sup>(1)</sup>                                                                                                                                                                                                                        | v3.3.0        | `true`                          |
+| `ODO_EXPERIMENTAL_MODE`    | Whether to enable experimental features. See [Experimental Mode](../user-guides/advanced/experimental-mode) for more details. Acceptable values: boolean values<sup>(1)</sup>                                                                                                                                                                                                  | v3.3.0        | `true`                          |
+| `ODO_PUSH_IMAGES`          | Whether to push the image once built; this is useful for integration tests running on podmanbecause images pushed by odo are created in a private registry and pulling them is not possible without setting up credentials to access it. `true` by default                                                                                                                     | v3.7.0        | `false`                         |
```

Comment:
Wondering if we need to provide so many details about the reasons..
From a user perspective, what's only useful IMO is knowing that this is read when applying image components as part of a Dev session running on Podman.

```suggestion
| `ODO_PUSH_IMAGES`          | Whether to push the images once built; this is used only when applying Devfile image components as part of a Dev Session running on Podman; this is useful for integration tests running on Podman. `true` by default                                                                                                                     | v3.7.0        | `false`                         |
```